pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = [
+    "setuptools",
+    "wheel",
+    "oldest-supported-numpy; python_version<'3.7'",
+    # we need at least 1.16.5 for a limited API bugfix
```

Comment:
Why == 1.16.6 when the comment and the code in setup.py say at least 1.16.5? Shouldn't it be == 1.16.5?

Reply:
I think my idea was to take the last patch version in that series, but probably doesn't matter much either way... I'll change to 1.16.5 for consistency.